nari/ext/act/utils.py

```diff
@@ -12,33 +12,42 @@
 
 
 # pylint: disable=invalid-name
-class ActLogChecksumType(Enum):
+class ActLogChecksumType(IntEnum):
     """List of hashsum algorithms used by different ACT versions"""
-    MD5 = "md5"
-    SHA256 = "sha256"
+    MD5 = 0
+    SHA256 = 1
```

Comment:
The more I think about this, the more I'm willing to block on not using `auto()` here. It's already unstable to keep two enums in both rust and python, and this is all to support a feature to read old act log files – which isn't a goal for this library.

Alternatively, I'll pass on this if we remove the checksum switching code in nari.

Reply:
MD5 has been dead long enough that IMO we can just support SHA256 exclusively. We should be using `auto()` on *all* `IntEnums` that don't need explicit values (due to gaps or whatever) tho.

Reply:
Changed it to now error if it sees it being the md5 file version and removed all the references to the algorithm due to MD5 being so old now that most people don't have logs of it